cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -752,8 +752,8 @@ runQueryRefScriptSizeCmd
         & onLeft left
 
 newtype RefInputScriptSize = RefInputScriptSize {refInputScriptSize :: Int}
-  deriving Generic
-  deriving anyclass ToJSON
+  deriving (Generic)
```

Comment:
We need to diagnose why this is happening. Something is up with your editor. Remove these non-semantic changes.

Reply:
this is being suggested when running `scripts/devshell/pretiffy `. I reverted it